content/asset-tracking/using-the-sdks.textile

```diff
@@ -295,6 +391,12 @@ subscriber.locations
     .launchIn(scope) // coroutines scope on which the locations are received
 ```
 
+```[swift]
+if let location = self.location {
```

Comment:
This doesn't do any subscription. Its just checking if location is not nil and printing it.

Reply:
Oops... missed the surrounding code 😊